Sources/AuthFoundation/Network/APIClient.swift

```diff
@@ -64,7 +64,7 @@ public protocol APIClient {
     func didSend<T>(request: URLRequest, received response: APIResponse<T>)
     
     /// Send the given URLRequest.
-    func send<T: Decodable>(_ request: URLRequest, parsing context: APIParsingContext?, completion: @escaping (Result<APIResponse<T>, APIClientError>) -> Void)
+    func send<T: Decodable>(_ request: URLRequest, parsing context: APIParsingContext?) async throws -> APIResponse<T>
```

Comment:
Is this a change to an external API? I guess this is then a major version bump?

Reply:
Yes this is a public API since it's used by the downstream libraries.  It's largely used as internal features of the SDK and while customers aren't expected to use this directly, it still needs to be public since Swift doesn't have any sort of "protected" accessibility option.

And definitely, this will be the SDK's first major release, because of the breaking API changes.